readme.txt

```diff
@@ -1,9 +1,9 @@
 === Microsoft Azure Storage for WordPress ===
-Contributors: 10up
+Contributors: msopentech, 10up, morganestes, stevegrunwell, lpawlik, ritteshpatel, johnwatkins0, rickalee
 Tags: Microsoft, Microsoft Open Technologies, Microsoft Azure, Microsoft Azure Storage, Media Files, Upload, CDN, blob storage
 Requires at least: 4.0
-Tested up to: 5.0
-Stable tag: 4.2.0
+Tested up to: 5.3
+Stable tag: 4.2.1
```

Comment:
Let's go with 4.3.0 as there's a decent amount going into this release, in terms of things previously merged into develop/master